src/pool/srv_pool.c

```diff
@@ -51,13 +51,25 @@
 uint32_t
 ds_pool_get_vos_df_version(uint32_t pool_global_version)
 {
-	if (pool_global_version >= 3)
+	if (pool_global_version == 4)
+		return VOS_POOL_DF_2_8;
+	if (pool_global_version == 3)
 		return VOS_POOL_DF_2_6;
 	else if (pool_global_version == 2)
 		return VOS_POOL_DF_2_4;
 	return 0;
 }
 
+/** Return the default VOS DF version for the default pool global version. */
```

Comment:
[Note] I think the first "default" in this comment should be deleted: One pool global version maps to only one VOS DF version; there's no such thing as default or non-default VOS DF version in this context.